src/doc/trpl/comments.md

```diff
@@ -1,47 +1,45 @@
 % Comments
 
-Now that we have some functions, it's a good idea to learn about comments.
+Now that we have some functions, it’s a good idea to learn about comments.
```

Comment:
I guess I'm just worried that we'll have to continually make passes through the documentation to update something like this, or I just don't know how to type it!